internal/fluentbit/config/builder/sections_config_builder.go

```diff
@@ -0,0 +1,129 @@
+package builder
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	telemetryv1alpha1 "github.com/kyma-project/telemetry-manager/apis/telemetry/v1alpha1"
+	"github.com/kyma-project/telemetry-manager/internal/fluentbit/config"
+	logpipelineutils "github.com/kyma-project/telemetry-manager/internal/utils/logpipeline"
+)
+
+var (
+	ErrInvalidPipelineDefinition = errors.New("invalid pipeline definition")
+)
+
+type PipelineDefaults struct {
+	InputTag          string
+	MemoryBufferLimit string
+	StorageType       string
+	FsBufferLimit     string
+}
+
+// FluentBit builder configuration
+type BuilderConfig struct {
```

Comment:
Do we need to export the `BuilderConfig` struct and its fields?
Also, would it make sense to move it to the `config_builder.go` file?